aziotctl/src/internal/check/checks/host_connect_iothub.rs

```diff
@@ -159,3 +166,81 @@ impl HostConnectIotHub {
         Ok(CheckResult::Ok)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::internal::check::{CheckerCache, CheckerCfg, CheckerShared, DaemonConfigs};
+
+    use super::HostConnectIotHub;
+    use aziot_identityd_config::{
+        Endpoints, ManualAuthMethod, Provisioning, ProvisioningType, Settings,
+    };
+    use std::path::PathBuf;
+
+    #[tokio::test]
+    async fn tls_handshake_timesout() {
+        let mut host = HostConnectIotHub::new(
+            "hostConnectIoTHubMQTT",
+            "
```

Comment:
Remove this newline inside the string literal.

Reply:
Done